bin/rucio

```diff
@@ -2184,6 +2187,7 @@ You can filter by key/value, e.g.::
     add_rule_parser.add_argument('--comment', dest='comment', action='store', help='Comment about the replication rule')
     add_rule_parser.add_argument('--ask-approval', dest='ask_approval', action='store_true', help='Ask for rule approval')
     add_rule_parser.add_argument('--asynchronous', dest='asynchronous', action='store_true', help='Create rule asynchronously')
+    add_rule_parser.add_argument('--when-to-inject', dest='when_to_inject', action='store', help='Time when to start applying the rule (UTC)')
```

Comment:
add example in the help how the user has to write the time

Reply:
Good idea. Added an example.